Formula/gromacs.rb

```diff
@@ -25,7 +25,10 @@ class Gromacs < Formula
   depends_on "gcc" # for OpenMP
   depends_on "openblas"
 
-  fails_with :clang
+  on_macos do
+    depends_on "libomp"
```

Comment:
This change may have succeeded in establishing a package dependency, but without additional CMake arguments, the build system configuration is still free to find `libgomp` if it is available and finds it preferable. I assume that is what happened in my test.